CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 * Added support for `np.float32` with `DissimilarityMatrix` objects.
+* Added support for method and number_of_dimensions to permdisp.
```

Comment:
```suggestion
* Added support for method and number_of_dimensions to permdisp reducing the runtime by over 10x at 2000 samples, [issue #1769](https://github.com/biocore/scikit-bio/pull/1769).
```

Reply:
...on issue 1768, it may be nice to note the observed performance differences with the default 10 dimensions